.github/PULL_REQUEST_TEMPLATE/bug-fix.md

```diff
@@ -0,0 +1,42 @@
+## Description
+<!--
+Thank you so much for your PR!  To help us review your contribution, please
+consider the following points:
+
+- The PR title should summarize the changes, for example "Add new group argument for the
+  pair plot".  Avoid non-descriptive titles such as "Addresses issue #348". If your pull 
+  request addresses an issue, please use the pull request title to describe 
+  the issue and mention the issue number in the pull request description.
+
+- The description should provide at least 1-2 sentences describing the pull request
+  in detail (Why is this change required?  What problem does it solve?) and
+  link to any relevant issues. If modifying a plot, render your plot to inspect for changes 
+  and copy image in the pull request message on Github
+
+- Please prefix the title of incomplete contributions with [WIP] (to indicate a work in 
+  progress). WIPs may be useful to (1) indicate you are working on something to avoid 
+  duplicated work, (2) request broad review of functionality or API, or (3) seek collaborators.
+-->
+
+
+## Checklist
+<!-- Feel free to remove check-list items aren't relevant to your change -->
+
+- [ ] Does the PR follow [official](https://github.com/arviz-devs/arviz/blob/master/CONTRIBUTING.md#pull-request-checklist) 
+      PR format?
+- [ ] Does the PR include new or updated tests to prevent issue recurrence (using [pytest fixture pattern](
+      https://docs.pytest.org/en/latest/fixture.html#fixture))?
+- [ ] Is your code style correct (follows pylint and black guidelines)?
+- [ ] Added the change in [change log](https://github.com/arviz-devs/arviz/blob/master/CHANGELOG.md) in correct format.
```

Comment:
In order to make the most out of having 3 PR templates, I would use something like `Is fix listed in the [Maintenance and fixes](https://github.com/arviz-devs/arviz/blob/master/CHANGELOG.md#maintenance-and-fixes) section of the changelog?`. 

Note that using the `#maintenance-and-fixes` will always link to the _first_ occurrence of "Maintenance and fixes" (the second is actually [`#maintenance-and-fixes-1`](https://github.com/arviz-devs/arviz/blob/master/CHANGELOG.md#maintenance-and-fixes-1) and so on). Unless there were some typo in the changelog section title the link will always point to the Unrelease section of the changelog. A similar thing can be done with the other sections and PR templates.

Reply:
@OriolAbril Okay, Should we also specify a format to add changes to `changelog`?